realtimetrains.py

```diff
@@ -145,90 +158,58 @@ def populate(self):
             return False
         else:
             self.webpage_checksum = md5sum
-
-        soup = BeautifulSoup(r.text, "html.parser")
-        if soup.text == NO_SCHEDULE:
+        
+        if 'realtimeActivated' not in r.json().keys():
             self.running = False
-            raise RuntimeError("schedule not found")
-        # Top of page shows schedule info, including if a
-        # runs-as-required train is active
-        schedule_info = soup.find("div",
-                                  attrs={"class": "detailed-schedule-info"})
-        # Text in schedule_info isn't tagged well
-        if "Running" in schedule_info.text:
+            #raise RuntimeError("schedule not found")
```

Comment:
I think we can get rid of this. In the future we should add error checking around every `requests.get` call to see if there's a 404 or it's been a server error or something.

Reply:
I'll use `r.raise_for_status` after the requests